docs/commands/enter.md

```diff
@@ -0,0 +1,24 @@
+# enter
+
+This command creates a new shell and begin at this path.
+
+## Examples
+
+```shell
+/home/foobar> cat user.json
+{
+	"Name": "Peter",
+	"Age": 30,
+	"Telephone": 88204828,
+	"Country": "Singapore"
+}
+/home/foobar> enter user.json
+/> ls
+━━━━━━━┯━━━━━┯━━━━━━━━━━━┯━━━━━━━━━━━
+ Name  │ Age │ Telephone │ Country
+───────┼─────┼───────────┼───────────
+ Peter │  30 │  88204828 │ Singapore
+━━━━━━━┷━━━━━┷━━━━━━━━━━━┷━━━━━━━━━━━
+/> exit
+/home/foobar>
```

Comment:
Looks good so far. You can also add an example of `enter`ing into another directory, so  you can work with multiple directories at the same time.